dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -193,6 +200,8 @@ export const enhanceCards = (
 		editionId?: EditionId;
 		containerPalette?: DCRContainerPalette;
 	},
+	cardInTagFront: boolean,
+	pageId?: string,
```

Comment:
I think that if your last parameter is an object, it should contain all the possible values that you want to discriminate on.

This is also [what the Deno style guide recommends](https://deno.land/manual@v1.34.3/references/contributing/style_guide#exported-functions-max-2-args-put-the-rest-into-an-options-object) and while we do not have a strict rule for this I think it should apply here.

Reply:
This is great, thanks both. Pushed the change